RiotSwiftUI/Modules/Room/LocationSharing/View/LocationSharingView.swift

```diff
@@ -84,6 +85,38 @@ struct LocationSharingView: View {
         .navigationViewStyle(StackNavigationViewStyle())
     }
     
+    var buttonsView: some View {
+        VStack(alignment: .leading, spacing: 15) {
+            if !context.viewState.isPinDropSharing {
+                LocationSharingOptionButton(text: VectorL10n.locationSharingStaticShareTitle) {
+                    context.send(viewAction: .share)
+                } content: {
+                    LocationSharingUserMarkerView(isMarker: false, avatarData: context.viewState.userAvatarData)
+                }
+                .disabled(!context.viewState.shareButtonEnabled)
+                // Disable for now until live location sharing is done
+                if BuildSettings.liveLocationSharingEnabled {
+                    LocationSharingOptionButton(text: VectorL10n.locationSharingLiveShareTitle) {
+                        // TODO: - Start live location sharing
+                    } content: {
+                        LocationSharingOptionButtonIcon(fillColor: Color.purple, image: Asset.Images.locationLiveIcon.image)
```

Comment:
We should use `Theme` colors. If it does not exist yet we should add a property.